policies/adding-new-role-members.md

```diff
@@ -13,21 +13,23 @@ which is highlighted in the process below.
 
 The process is:
 
-1. A Coordination Committee member sends a message to the person
-   nominated, making sure they understand the role, its responsibilities, and
-   confirming that the nominee agrees. 
+1. A voting member or maintainer in a related role sends a message to the person nominated (with the CoCo in the cc), making sure they
+   understand the role, its responsibilities, and confirming that the nominee agrees. 
```

Comment:
```suggestion
1. A voting member or maintainer in a related role sends a message to the person nominated
   (with the Coordination Committee in the cc), making sure they understand the role, 
   its responsibilities, and confirming that the nominee agrees. 
```
should spell out here as below since this is the first occurrence